### PR TITLE
Adding generated azblob code

### PR DESCRIPTION
**Pull request by catalinaperalta**
This PR removes the sample code generated for storage and adds the generated code for blob storage

### CODE REVIEW

File: sdk/azidentity/go.sum [hunk 1]
```diff
@@ -1,3 +1,8 @@
+github.com/Azure/azure-sdk-for-go v40.5.0+incompatible h1:CVQNKuUepSFBo6BW6gM1J9slPHLRcjn6vaw+j+causw=
```
Comment (jhendrixMSFT) [inline sdk/azidentity/go.sum]: Run `go mod tidy` to clean up spurious entries.

File: sdk/azidentity/aad_identity_client.go [hunk 2]
```diff
@@ -141,7 +142,11 @@ func (c *aadIdentityClient) createAccessToken(res *azcore.Response) (*azcore.Acc
 		ExpiresIn json.Number `json:"expires_in"`
 		ExpiresOn string      `json:"expires_on"`
 	}{}
-	if err := json.Unmarshal(res.Payload, &value); err != nil {
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return nil, err
+	}
+	if err := json.Unmarshal(body, &value); err != nil {
```
Comment (jhendrixMSFT) [inline sdk/azidentity/aad_identity_client.go]: You can replace `json.Unmarshal()` with `res.UnmarshalAsJSON()`.  You won't have to call `ioutil.ReadAll()` thus avoiding an extra copy of the underlying data.

File: sdk/azidentity/azidentity.go [hunk 3]
```diff
@@ -164,8 +169,9 @@ func newDefaultMSIPipeline(o ManagedIdentityCredentialOptions) azcore.Pipeline {
 	var statusCodes []int
 	// retry policy for MSI is not end-user configurable
 	retryOpts := azcore.RetryOptions{
-		MaxTries:   5,
+		MaxRetries: 5,
```
Comment (jhendrixMSFT) [inline sdk/azidentity/azidentity.go]: Note this changed from tries to retries.  Leaving this as 5 will mean a total of 6 tries.  Is this expected, or should the value change to 4?

File: sdk/storage/blob/2019-07-07/azblob/go.mod [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module github.com/Azure/azure-sdk-for-go/sdk/storage/2019-07-07/azblob
```
Comment (jhendrixMSFT) [inline sdk/storage/blob/2019-07-07/azblob/go.mod]: `github.com/Azure/azure-sdk-for-go/sdk/storage/blob/2019-07-07/azblob`

File: sdk/storage/blob/2019-07-07/azblob/go.sum [hunk 5]
```diff
@@ -0,0 +1,5 @@
+github.com/Azure/azure-sdk-for-go v40.5.0+incompatible h1:CVQNKuUepSFBo6BW6gM1J9slPHLRcjn6vaw+j+causw=
```
Comment (jhendrixMSFT) [inline sdk/storage/blob/2019-07-07/azblob/go.sum]: `go mod tidy`

File: sdk/azidentity/azidentity.go (hunk 3 above)
Comment (catalinaperalta) [reply sdk/azidentity/azidentity.go]: Yes, you're right this needs to be changed to 4.

File: sdk/azidentity/managed_identity_client.go [hunk 6]
```diff
@@ -129,10 +139,10 @@ func (c *managedIdentityClient) sendAuthRequest(ctx context.Context, msiType msi
 func (c *managedIdentityClient) createAccessToken(res *azcore.Response) (*azcore.AccessToken, error) {
 	value := struct {
 		// these are the only fields that we use
-		Token        string      `json:"access_token"`
-		RefreshToken string      `json:"refresh_token"`
-		ExpiresIn    json.Number `json:"expires_in"` // this field should always return the number of seconds for which a token is valid
-		ExpiresOn    string      `json:"expires_on"` // the value returned in this field varies between a number and a date string
+		Token        string         `json:"access_token,omitempty"`
+		RefreshToken string         `json:"refresh_token,omitempty"`
+		ExpiresIn    *wrappedNumber `json:"expires_in,omitempty"` // this field should always return the number of seconds for which a token is valid
```
Comment (jhendrixMSFT) [inline sdk/azidentity/managed_identity_client.go]: I'm guessing that now with `wrappedNumber` this no longer needs to be a pointer?

File: sdk/azidentity/aad_identity_client.go [hunk 7]
```diff
@@ -163,7 +164,11 @@ func (c *aadIdentityClient) createRefreshAccessToken(res *azcore.Response) (*tok
 		ExpiresIn    json.Number `json:"expires_in"`
 		ExpiresOn    string      `json:"expires_on"`
 	}{}
-	if err := json.Unmarshal(res.Payload, &value); err != nil {
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return nil, err
+	}
+	if err := json.Unmarshal(body, &value); err != nil {
```
Comment (jhendrixMSFT) [inline sdk/azidentity/aad_identity_client.go]: `res.UnmarshalAsJSON()`

File: sdk/azidentity/aad_identity_client.go [hunk 8]
```diff
@@ -289,7 +294,11 @@ func (c *aadIdentityClient) createUsernamePasswordAuthRequest(tenantID string, c
 
 func createDeviceCodeResult(res *azcore.Response) (*deviceCodeResult, error) {
 	value := &deviceCodeResult{}
-	if err := json.Unmarshal(res.Payload, &value); err != nil {
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return nil, err
+	}
+	if err := json.Unmarshal(body, &value); err != nil {
```
Comment (jhendrixMSFT) [inline sdk/azidentity/aad_identity_client.go]: `res.UnmarshalAsJSON()`

File: sdk/azidentity/azidentity.go [hunk 9]
```diff
@@ -72,7 +73,11 @@ func (e *AuthenticationFailedError) Error() string {
 
 func newAADAuthenticationFailedError(resp *azcore.Response) error {
 	authFailed := &AADAuthenticationFailedError{}
-	err := json.Unmarshal(resp.Payload, authFailed)
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+	err = json.Unmarshal(body, authFailed)
```
Comment (jhendrixMSFT) [inline sdk/azidentity/azidentity.go]: `res.UnmarshalAsJSON()`

File: sdk/azidentity/managed_identity_client.go [hunk 10]
```diff
@@ -128,16 +139,20 @@ func (c *managedIdentityClient) sendAuthRequest(ctx context.Context, msiType msi
...
+		ExpiresOn    string         `json:"expires_on,omitempty"` // the value returned in this field varies between a number and a date string
 	}{}
-	if err := json.Unmarshal(res.Payload, &value); err != nil {
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return nil, err
+	}
+	if err := json.Unmarshal(body, &value); err != nil {
```
Comment (jhendrixMSFT) [inline sdk/azidentity/managed_identity_client.go]: `res.UnmarshalAsJSON()`

**Pull request merged**